packages/react/src/lib/providers/i18n/i18n-provider-defaults.ts

```diff
@@ -52,6 +52,9 @@ export const defaultTranslations = {
     failedToLoadOptions: "Failed to load options",
     retry: "Retry",
   },
+  toc: {
+    search: "Search",
```

Comment:
should they provide this value to the ToC itself? It could be a different message depending on the use

Reply:
they can provide a custom placeholder, this is the placeholder by default